src/lib/forms/rjsf/RJSFCustomTemplates.js

```diff
@@ -236,6 +237,7 @@ ObjectFieldTemplateWrapperGrid.defaultProps = {};
  */
 export function ArrayFieldTemplateWithWrapper(props) {
   const { children, uiSchema } = props;
+  ignoreRJSFEnterEvent(document);
```

Comment:
Comment: Hmm I understand that we need to add this here because at the form cmp level these are not loaded as we can add new fields dynamically.

The only issue I can see is that in terms of performance this is worse as it will loop over items that were already processed. I would rather set the `onkeydown` when creating it, but in a first look it doesn't seem easy to do with rjsf ... However it's probably kicking in only when working on array fields so it should be good enough IMHO.